src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Figure implements AreaComputable {
+    private static final FigureShape FIGURE_SHAPE = FigureShape.CIRCLE;
+    private static final double RADIUS = 10;
+    private static final Color color = Color.WHITE;
+
+    public Figure() {
+
+    }
+
+    @Override
+    public String toString() {
+        return String.format("Figure: %s, area: %f sq.units, color: %s",
+                FIGURE_SHAPE.name().toLowerCase(), getArea(), color);
+    }
```

Comment:
<img width="827" alt="image" src="https://user-images.githubusercontent.com/58032907/193250007-1cc0f772-22cc-4f30-9f56-cf32da60eda7.png">

Reply:
i read it wrong, oops